profile/README.md

```diff
@@ -1,32 +1,43 @@
-# Arcalot
-
-<img align="right" width="200px" style="padding-left: 2em;" alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the silhouette of a castle in the background" src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png">
-
-The Arcalot community develops tools, plugins, and libraries that you can use either standalone as a library, and/or via a user interface or CLI. You can run the tools locally, remotely, or as part of a bigger system. Arcalot:
-
-* Helps you create workflows with **normalized input and output schemas**
-* Provides you with **assisted and automated root cause analysis** for the workflows you create as well as CI and other log systems
-* Provides **stable plugins for several workloads**
+# <a href="https://arcalot.io">Arcalot</a>
+<a href="https://arcalot.io"><img align="right" width="200px" style="padding-left: 2em;"
+ alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the 
+ silhouette of a castle in the background" 
+ src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png"></a>
+
+The Arcalot community is dedicated to developing modular tools, plugins, and libraries
+with flexible implementations independently or as complete end-to-end solutions. We
+believe in enabling the automation and portability of complex tasks and in
+pre-validating actions to avoid costly re-runs due to late failures and incompatible
+data.
 
 <br clear="right"/>
 
-## Arcaflow
+## <a href="https://arcalot.io/arcaflow">Arcaflow</a>
 
-<img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png">
+<a href="https://arcalot.io/arcaflow"><img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png"></a>
 
-Arcaflow is a workflow engine consisting of three main components:
+Arcaflow is a workflow system consisting of three main components:
```

Comment:
Not sure "workflow system" has much intrinsic meaning at this point. Both words are way too generic without context. Maybe even drop "workflow" here. You follow with a list of three components, followed by describing creation of workflows referencing the plugins...

Or even just "Arcaflow consists of three main components"

Reply:
I don't like using "engine" here because that is the name of a component of the system as a whole. As an entry point to the project, I think it's important to be clear up front that this is a workflow.... system? I'm not sure what other noun would make better sense here.

Reply:
Yes, "engine" is one of the Arcaflow components, and losing that is an improvement. But "workflow system" doesn't make any sense -- at best it's ambiguous, redundant, and a bit tedious. We go into the details, starting with the list. That's quite enough!

Reply:
I don't find "workflow system" overly ambiguous for an introductory mostly marketing language page. I feel that Arcaflow needs a short descriptive definition of what it is, and I'm open to other ideas, but I think that there needs to be *something* here.

Reply:
A _workflow_ is a series or a description of a series of tasks which are performed in sequence; a _system_ is...well, it could be anything, really...but here I think it's a framework or structure which is intended to support or perform some purpose; so, we're missing the word which describes how the _system_ "does" the _workflow_.  I would propose something like "a workflow execution system".

"Jenkins is an [...] automation software DevOps tool", perhaps Arcaflow can be something similar.

Reply:
workflow orchestration system?